### PR TITLE
AlexStefan add Xamarin

### PR DESCRIPTION
**Pull request by AlexStefan**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/xamarin/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+aliases: mono
+created_by: Alexandru Stefan Popescu
```
Comment (nayafia) [inline topics/xamarin/index.md]: This should display the authors of Xamarin (`Miguel de Icaza, Nat Friedman`), not your name

File: topics/xamarin/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+aliases: mono
```
Comment (nayafia) [inline topics/xamarin/index.md]: Maybe this should be a related topic, not an alias? (unless Mono and Xamarin are exactly the same)

File: topics/xamarin/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+created_by: Alexandru Stefan Popescu
+display_name: Xamarin
+github_url: https://github.com/xamarin
+logo: xamarin.png
+released: May 16 2011
+short_description: The Xamarin platform that allow you to develop applications for iOS and Android.
```
Comment (nayafia) [inline topics/xamarin/index.md]: Suggested rephrase:

> Xamarin is a platform for developing iOS and Android applications.

File: topics/xamarin/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+topic: xamarin
+url: https://www.xamarin.com/
+wikipedia_url: https://en.wikipedia.org/wiki/Xamarin
+---
+The Xamarin platform consists of a number of elements that allow you to develop applications for iOS and Android:
```
Comment (nayafia) [inline topics/xamarin/index.md]: Could you condense this down a bit to just 1-3 sentences?

File: topics/xamarin/index.md (hunk 2 above)
Comment (AlexStefan) [reply topics/xamarin/index.md]: Moved it as related.

File: topics/xamarin/index.md (hunk 3 above)
Comment (AlexStefan) [reply topics/xamarin/index.md]: Done, thank you for the suggestion.

File: topics/xamarin/index.md (hunk 4 above)
Comment (AlexStefan) [reply topics/xamarin/index.md]: Changed it. I'd like to use something that's already out there and not to create some sentences that may miss something or not cover everything.

**Comment by cheshire137** [comment]
Looks like AlexStefan made some updates since your last review, @nayafia! 👀

**Comment by nayafia** [comment]
Thanks @AlexStefan! 🎉

**Pull request merged**